README.md

```diff
@@ -22,16 +22,16 @@ This project allows the [Community Tech bot](https://en.wikipedia.org/wiki/User:
 ## Installation
 Virtualenv is recommended:
 
-    virtualenv venv
+    python3 -m venv venv
```

Comment:
I think `venv` is duplicated here?

Reply:
The first one is module name, the second one is destination directory.

Reply:
I had no idea. Thanks!